package.json

```diff
@@ -16,6 +16,7 @@
 		"react-bootstrap": "^2.10.5",
 		"react-dom": "^18.3.1",
 		"react-hot-toast": "^2.4.1",
+		"react-icons": "^5.3.0",
```

Comment:
YASSSSS! I love the use of icons while keeping the button colors the same for smaller screen sizes!